### ISSUE TITLE
Plugin stuck on "downloading wakatime-cli"

### ISSUE THREAD

**Issue by LuudJacobs** [wakatime/atom-wakatime#101]
I noticed Wakatime didn't track my recent coding activity, and found that in the status bar the plugin is stuck on "downloading wakatime-cli". I didn't see any errors in the atom console or in the Wakatime log files.

**Comment by JobJob** [comment]
Yeah I'm getting this basically every time I restart or reopen Atom - sometimes it also stalls at "extracting ...". 

It also seems to stop tracking after a day or so - not sure exactly what the conditions are for this.

Uninstalling and reinstalling the plugin seems to fix it for a bit, but obviously having to do this often is not ideal.

**Comment by jfwilkus** [comment]
I've been having the same problem for months. I've enabled debugging but it doesn't show anything.  

For instance I know I worked quite a few hours on a project on Monday but Waka didn't log any of it. Grepping the `wakatime.log` for the project doesn't find anything for that day.

Sometimes restarting the project works. Sometimes reinstalling the plugin works.

It's pretty inconsistent, I've not been able to see a pattern as to when it works and when it doesn't.

Atom 1.50.0
Waka 9.0.2
